README.md

```diff
@@ -50,9 +50,9 @@ rule should be able to be reasoned with on its own.
 ## Input data
 
 This build starts by downloading sequences from
-[https://data.nextstrain.org/files/zika/sequences.fasta.xz](data.nextstrain.org/files/zika/sequences.fasta.xz)
+[https://data.nextstrain.org/files/zika/sequences.fasta.xz](https://data.nextstrain.org/files/zika/sequences.fasta.xz)
 and metadata from
-[https://data.nextstrain.org/files/zika/metadata.tsv.gz](data.nextstrain.org/files/zika/metadata.tsv.gz).
+[https://data.nextstrain.org/files/zika/metadata.tsv.gz](https://data.nextstrain.org/files/zika/metadata.tsv.gz).
```

Comment:
Looks like @tsibley caught this earlier in https://github.com/nextstrain/zika/pull/15#discussion_r832726121 with a different suggestion.

Reply:
Thanks @victorlin and @tsibley! Yes, auto-linking base URLs is more streamlined.